config/log4j2.properties

```diff
@@ -0,0 +1,82 @@
+status = error
```

Comment:
It's been slightly modified but I reduced it to only log to console. Now using official docker  image configuration (https://github.com/docker-library/elasticsearch/blob/1107caed5c403a63d99c1b1ae1f7002d387c7e85/5/config/log4j2.properties).